examples/step-5/step-5.cc

```diff
@@ -113,6 +113,14 @@ double coefficient (const Point<dim> &p)
   else
     return 1;
 }
+// @note It looks like we use a discontinuous coefficient, however,
+// the finite element method using quadrature really sees only a
+// polynomial approximation (defined by a
+// <code>quadrature_formula</code>) of that discontinuous coefficient
+// on every mesh. (Though that polynomial approximation of course
+// tends to a discontinuous function under mesh refinement.) Refer to
+// CellAccessor::set_material_id to set a discontinuous coefficient
```

Comment:
@jppelteret Sorry, no. It is just about an inner interface, a very simple question how to define it correctly. The provided way can lead to a somehow unexpected result due to the interplay between coefficient (as defined by points) and integration by a quadrature. Added a link to the step-28 as an advanced example of usage of material_id.